backends-velox/src/test/scala/io/glutenproject/expression/VeloxUdfSuite.scala

```diff
@@ -82,7 +82,8 @@ class VeloxUdfSuiteLocal extends VeloxUdfSuite {
   override val master: String = "local[2]"
   override protected def sparkConf: SparkConf = {
     super.sparkConf
-      .set("spark.gluten.sql.columnar.backend.velox.udfLibraryPaths", udfLibPath)
+      .set("spark.files", udfLibPath)
+      .set("spark.gluten.sql.columnar.backend.velox.udfLibraryPaths", "libmyudf.so")
```

Comment:
Please do not hard code "libmyudf.so" here. Use a helper function to extract the filename from `udfLibPath`. Note the `udfLibPath` can also be a comma-separated string with multiple paths.